docs/dev/create-an-integration-recommended-monitor.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Create an Integration Recommended Monitor
+kind: documentation
+---
+
+## Overview
+
+[Datadog Monitors][1] enable you to efficiently monitor your infrastructure and integrations by tracking key metrics. Datadog provides a set of out-of-the-box monitors for many features and integrations. You can access these by viewing your [Monitors Template list][4].
+
+If you have [created a Datadog integration][2], you may wish to create an out-of-the-box monitor to help your integration's users more quickly find value in your integration. This guide provides steps for creating an integration recommended monitor and best practices to follow during the creation process.
+
+To create a Datadog integration, see [Create a New Integration][2].
+
+
+## Create an integration recommended monitor
+### Create a new monitor
+
+In Datadog, [create a new monitor][4]. 
+
+Follow the best practices in this guide when defining your monitor.
+
+### Export your monitor
+
+Select the checkbox to export the monitor as a recommended monitor.
+Export your monitor to JSON using the export button.
```

Comment:
should we add "click checkbox to export in recommended monitor schema"?

Reply:
Is the line above this not sufficient?

Reply:
I don't know how I missed that 🤦‍♂️ my bad